### ISSUE TITLE
Translation of menu-item-css-classes

### ISSUE THREAD

**Issue by Pasley70** [iionly/celebrations#2]
The plugin translate the site-menu-item

english:

```
<li class="elgg-menu-item-celebrations">
```

german: 

```
<li class="elgg-menu-item-jahrestage-anzeigen">
```

which is not dramatic. But - for example - I put icons via CSS before the menu items, so that I need a CSS-class for each language. A single class would may be better e.g. for individual themes.

**Comment by iionly** [comment]
It hasn't been intended. The different class names are caused by using elgg_echo() for the menu identifier by mistake instead of just using a const string identifier.

If you want to correct it before I will be able to release a fixed version, you can change it in the init function in start.php:

``` php
if (elgg_is_logged_in()) {
    elgg_register_menu_item('site', array('name' => 'celebrations', 'text' => elgg_echo('celebrations:shorttitle'), 'href' => "celebrations/celebrations"));
}
```

**Comment by Pasley70** [comment]
Thanks for the hint!

**Comment by iionly** [comment]
Fixed in release 1.9.5.
